backend/config/firebase.py

```
@@ -0,0 +1,14 @@
+import os
+import firebase_admin
```

Ruff (0.12.2)

`firebase_admin` imported but unused

Remove unused import: `firebase_admin`

(F401)